### ISSUE TITLE
Create a Demo-User Button

### ISSUE THREAD

**Issue by lyjimmy1** [Team-YEAH/venmoClone#15]

**Comment by lyjimmy1** [comment]
after clicking on the button, the user should have full accessibility to the webiste
